ArduCopter/Parameters.cpp

```diff
@@ -429,14 +429,12 @@ const AP_Param::Info Copter::var_info[] = {
     // @Description: Converts pilot roll and pitch into a desired rate of rotation in ACRO and SPORT mode.  Higher values mean faster rate of rotation.
     // @Range: 1 10
     // @User: Standard
-    GSCALAR(acro_rp_p,                 "ACRO_RP_P",           ACRO_RP_P),
 
     // @Param: ACRO_YAW_P
     // @DisplayName: Acro Yaw P gain
     // @Description: Converts pilot yaw input into a desired rate of rotation.  Higher values mean faster rate of rotation.
     // @Range: 1 10
```

Comment:
Shouldn't you also remove the parameter metadata comments?

Reply:
Agreed, we should remove old param metadata.

Reply:
I understand that Randy leaves them in. I will see what he says.

Reply:
its safe to delete now we have param versioning on the wiki I think.

Reply:
let's leave it in until we have confirmation from MP.